OptimizelySDK/OptimizelyFactory.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using OptimizelySDK.Bucketing;
+using OptimizelySDK.Config;
+using OptimizelySDK.ErrorHandler;
+using OptimizelySDK.Event.Dispatcher;
+using OptimizelySDK.Logger;
+
+namespace OptimizelySDK
+{
+
+    /// TODO: Need to check with Mike, no need of this class 
```

Comment:
This is not the case anymore right?

Reply:
Yes, will add unit test and more convenience methods in separate PR. Removing this comment.